services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str | None = None,
+                first_name: str | None = None,
+                last_name: str | None = None) -> None:
+    new_user = User.objects.create_user(username=username, password=password)
```

Comment:
it is better to use get_user_model() method instead of User
fix this everywhere

Reply:
Done